st2actions/st2actions/notifier/notifier.py

```diff
@@ -231,10 +226,18 @@ def _get_trace_context(self, execution_id):
         return None
 
     def _post_generic_trigger(self, liveaction_db=None, execution_db=None):
-        if not ACTION_SENSOR_ENABLED:
+        if not cfg.CONF.action_sensor.enable:
             LOG.debug('Action trigger is disabled, skipping trigger dispatch...')
             return
 
+        target_statuses = cfg.CONF.action_sensor.emit_when
```

Comment:
Would be a good idea to log a message under DEBUG if trigger is not emitted due to the status not being in ``emit_when` `list.

Reply:
Will do. Thanks for the comments!